src/fiatconnect/ReviewScreen.tsx

```diff
@@ -416,7 +418,8 @@ function getDisplayAmounts({
   } else {
     const receive = normalizedQuote.getCryptoAmount()
     const total = normalizedQuote.getFiatAmount()
-    const fee = normalizedQuote.getFeeInFiat(exchangeRates, tokenInfo)
+    const fee =
+      (!!tokenInfo && normalizedQuote.getFeeInFiat(exchangeRates, tokenInfo)) || new BigNumber(0)
```

Comment:
```suggestion
      tokenInfo ? normalizedQuote.getFeeInFiat(exchangeRates, tokenInfo) : new BigNumber(0)
```
nit: this is a bit clearer imo

Reply:
nvm, this messes up the types since it's not actually the same